<ui version="4.0" >
 <class>SaveItemsDialog</class>
 <widget class="QDialog" name="SaveItemsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Save Changes</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="msgLabel" >
     <property name="text" >
      <string>Save the changes of the following items:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget" />
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveItemsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
